.NET/Microsoft.Recognizers.Text.DateTime/Parsers/BaseDurationParser.cs

```diff
@@ -295,6 +295,12 @@ private DateTimeResolutionResult ParseImplicitDuration(string text, DateObject r
                 ret = result;
             }
 
+            if ((config.Options & DateTimeOptions.CalendarMode) != 0 &&
```

Comment:
Can you add a code comment that explains the purpose of the check? Like in the next if.